packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -293,31 +236,40 @@ const Toolbar = memo(
   }
 );
 
+function ToolbarButtons(props: { items: React.ReactNode[] }) {
+  return (
+    <>
+      {props.items.map((el, i) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <div key={i} className={styles.item}>
+          {el}
+          {i !== props.items.length - 1 && <div className={styles.divider} />}
+        </div>
+      ))}
+    </>
+  );
+}
+
 function FocusOnSubtree({
   onFocusOnSubtree,
   selectedNode,
 }: {
   selectedNode: ProfileHeaderProps['selectedNode'];
   onFocusOnSubtree: ProfileHeaderProps['onFocusOnSubtree'];
 }) {
-  const onClick = selectedNode.mapOr(
-    () => {},
-    (f) => {
-      return () => onFocusOnSubtree(f.i, f.j);
-    }
-  );
-
   return (
     <Tooltip placement="top" title="Collapse nodes above">
       <div>
-        <Button
+        <GButton
           disabled={!selectedNode.isJust}
-          onClick={onClick}
-          className={styles.collapseNodeButton}
+          onClick={() => {
+            if (selectedNode.isJust) {
```

Comment:
This may be a bit of a preference, but I think it's better to use as minimal API surface as possible so we don't have to lear too many libraries. This also probably has the same effect as just assigning empty function in case `!selectNode.isJust` because at the same time the button is disabled in that case.

Reply:
Yeah that's fine, historically it was used when typescript hasn't been fully added, so it was a way to force people to deal with undefined cases.